src/routes/safe/components/Settings/SpendingLimit/FormFields/ResetTime.tsx

```diff
@@ -81,17 +82,30 @@ const ResetTimeOptions = styled.div`
   grid-area: resetTimeOption;
 `
 
-export const RESET_TIME_OPTIONS = [
-  { label: '1 day', value: '1' },
-  { label: '1 week', value: '7' },
-  { label: '1 month', value: '30' },
+const RESET_TIME_OPTIONS = [
+  { label: '1 day', value: '1440' }, // 1 day x 24h x 60min
+  { label: '1 week', value: '10080' }, // 7 days x 24h x 60min
+  { label: '1 month', value: '43200' }, // 30 days x 24h x 60min
 ]
 
+const RINKEBY_RESET_TIME_OPTIONS = [
+  { label: '5 minutes', value: '5' },
+  { label: '30 minutes', value: '30' },
+  { label: '1 hour', value: '60' },
```

Comment:
@francovenica @liliya-soroka are you OK with having only small timers on Rinkeby?